dbt_coverage/__init__.py

```diff
@@ -168,6 +230,10 @@ def _normalize_column_names(columns):
             col['name'] = col['name'].lower()
         return {col['name']: col for col in columns.values()}
 
+    @staticmethod
+    def _normalize_path(path: str) -> str:
+        return str(PurePath(path).as_posix())
```

Comment:
@mrshu Upon reflection I have swapped `PurePath` for `Path` because; 1. I cannot say that we may _never_ want to use I/O operations in the future and 2. it removes the need to import another module.